sis_scraper/main.py

```diff
@@ -286,7 +287,7 @@ def toTitle(text):
             for section in course["sections"]:
                 crn_to_courses[section["crn"]] = course["id"]
 
-                conflict = [0] * (64 * 3)
+                conflict = [0] * (64 * 4)
                 for time in section["timeslots"]:
                     for day in time["days"]:
                         for i in range(500, 2400, 100):
```

Comment:
It wont let me make a comment on the line, but line 314 is now also wrong. I think it needs to be `f"\t{crn}u32 => [{int(conflict[:64], 2)}, {int(conflict[64:128], 2)}, {int(conflict[128:192], 2)}, {int(conflict[192:], 2)}],\n"`

Reply:
That will cause issues with quacs-rs which expects only three integers there.  For example, line 308 will need to be updated to `pub static CRN_TIMES: Map<u32, [u64; 4]> = phf_map! {`.

Reply:
The conflicts goes from 5am to midnight daily.  If we cut that down to 7am to 11pm, we'll be able to exactly fit 6 days of conflicts in 3 64 bit numbers (unless I messed up the math and it's short by a bit or two).

Reply:
(That would be changing this line from 500 and 2400 as the bounds to 700 and 2300)

Reply:
This no longer needs to be done